content/articles/how-to-create-a-rest-api-using-spring-boot-maven-and-mysql/index.md

```diff
@@ -0,0 +1,810 @@
+* Next we created a no-argument constructor, setters, and getters for our field, and override the toString,  equals, and hashCode methods.
+
+
+### 7. Adding Additional Maven Dependencies
+
+In this section we are going to add additional deficiencies to the project.Go to [***Maven Repository***](https://mvnrepository.com/) search for the following dependencies and add them to the dependencies section of the pom.xml file:
+* **Javax validation:** It is used to validate the fields of an object before persisting to in the database
+* **Io springfox:** A dependency that will allow us document our APIs.
+* Sync the newly added dependencies to the application.
+* Your modified pom.xml should like this:
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.5.2</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.lekwacious</groupId>
+    <artifactId>employee_app</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>employee_app</name>
+    <description>Demo project for Spring Boot</description>
+    <properties>
+        <java.version>11</java.version>
+    </properties>
+    <dependencies>
+        <!--        newly added dependencies-->
+        <dependency>
+            <groupId>javax.validation</groupId>
+            <artifactId>validation-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-boot-starter</artifactId>
+            <version>3.0.0</version>
+        </dependency>
+<!--        Already existing dependencies-->
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+            </plugin>
+        </plugins>
+    </build>
+
+</project>
+
+
+```
```

Comment:
Instead of writing the whole file here, just type out the modified part.

The whole file can be added as a link linking to a GitHub repository